src/platform/site-wide/loading-button/LoadingButton.jsx

```diff
@@ -31,6 +27,8 @@ export default function LoadingButton({
       {...props}
       disabled={isLoading || disabled}
       onClick={onClick}
+      // eslint-disable-next-line jsx-a11y/no-interactive-element-to-noninteractive-role
```

Comment:
Sort of concerned that I have to ignore this rule, but I'll defer to VSP and the accessibility folks

Reply:
Have you tried adding the <button> inside a <div>?

Reply:
@1Copenut Could we have a couple of eyes here?